js/web/lib/onnxjs/backends/webgl/ops/resize-packed.ts

```diff
@@ -241,8 +241,8 @@ const prepareInputs = (inputs: Tensor[], attributes: UpsampleAttributes): [reado
 
 const parseScalesData = (scale: Tensor, mode: string, isResize: boolean): number[] => {
   const scales = Array.from(scale.floatData);
-  scalesValidation(scales, mode, isResize);
-  return scales;
+  scalesValidation(scales as number[], mode, isResize);
+  return scales as number[];
```

Comment:
we can keep onnxjs code unchanged as they are going to deprecated. we don't plan to support new features to webgl.

Reply:
let me try to make some modifications to unit test so that it no longer depends on onnxjs/tensor. Then you don't need to modify any file under onnxjs folder .

Reply:
#19358 should work with it